pimd/pim_nb_config.c

```diff
@@ -2723,7 +2745,23 @@ int lib_interface_gmp_address_family_query_interval_modify(
 		change_query_interval(ifp->info, query_interval);
 	}
 #else
-	/* TBD Depends on MLD data structure changes */
+	struct pim_interface *pim_ifp;
+
+	switch (args->event) {
+	case NB_EV_VALIDATE:
+	case NB_EV_PREPARE:
+	case NB_EV_ABORT:
+		break;
+	case NB_EV_APPLY:
+		ifp = nb_running_get_entry(args->dnode, NULL, true);
+		pim_ifp = ifp->info;
+		if (!pim_ifp)
+			return NB_ERR_INCONSISTENCY;
+
+		query_interval = yang_dnode_get_uint16(args->dnode, NULL);
+		pim_ifp->gm_default_query_interval = query_interval;
```

Comment:
I've kept it separate because it's not clear yet what to do with the units (seconds/deciseconds/milliseconds). Also thought it's cleaner this way with a separate MLD/IPv6 block everywhere (for consistency).